todo.md

```
@@ -0,0 +1,99 @@
+- [ ] Create reset button to clear uploaded images and preferences.
+- [ ] Warn users before navigating away without downloading.
+
+---
+
+## 🌐 **Localization**
+- [ ] Add English and Japanese language support.
+- [ ] Store language preferences in LocalStorage.
+
+---
+
+## ♿ **Accessibility**
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)